book/visual-effects.md

```diff
@@ -330,8 +330,8 @@ class Browser:
 ```
 
 Next, we need to copy the data to an SDL surface. This requires
-telling SDL what order the pixels are stored in and your computer's
-[endianness][wiki-endianness]:
+telling SDL what order the pixels are stored in, which depends
```

Comment:
I don't think this change improves anything much.

Reply:
Ok